tests/integration.spec.ts

```diff
@@ -120,7 +121,7 @@ test('should not duplicate gitignore entries', async ({ run, dir }) => {
 
 test('should install with "npm ci" in GHA when using npm with package-lock enabled', async ({ dir, run, packageManager }) => {
   test.skip(packageManager !== 'npm');
-  
```

Comment:
I think we should add automatic formatting to this repository to reduce noise from changes like this one.